src/main/java/com/bamdoliro/gati/domain/board/domain/Board.java

```diff
@@ -35,6 +33,11 @@ public class Board extends BaseTimeEntity {
     @Column(name = "content", columnDefinition = "TEXT", length = 4000, nullable = false)
     private String content;
 
+    @Enumerated(EnumType.STRING)
+    @Column(name = "status", nullable = false)
+    @Setter
```

Comment:
setter annotation을 바로 사용하는 것보다 `deleteBoard` method를 직접 써 주는 게 좋을 것 같습니다.
```suggestion
    public void deleteBoard() {
        this.status = Status.DELETED;
    }
```

Reply:
왜냐하면 setter가 있으면 deleted board 에서 -> EXISTED로 바뀔 수 있기 때문에.